parlai/crowdsourcing/tasks/chat_demo/run.py

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import os
+from dataclasses import dataclass, field
+
+import hydra
+from omegaconf import DictConfig
+
+# From the Mephisto repo
+from examples.parlai_chat_task_demo.parlai_test_script import (
+    TestScriptConfig,
+    TASK_DIRECTORY,
+)
+from mephisto.core.hydra_config import register_script_config
+from mephisto.core.operator import Operator
+from mephisto.server.blueprints.parlai_chat.parlai_chat_blueprint import (
+    SharedParlAITaskState,
+)
+from mephisto.utils.scripts import load_db_and_process_config
+
+from parlai.crowdsourcing.utils.mturk import MTurkRunScriptConfigMixin
+
+
+@dataclass
+class ScriptConfig(MTurkRunScriptConfigMixin, TestScriptConfig):
+    """
+    Use the mixin's soft-blocking with the defaults from the Mephisto version.
+    """
+
+    monitoring_log_rate: int = field(
+        default=30,
+        metadata={
+            'help': 'Frequency in seconds of logging the monitoring of the crowdsourcing task'
+        },
+    )
+
+
+register_script_config(name="scriptconfig", module=ScriptConfig)
+relative_task_directory = os.path.relpath(TASK_DIRECTORY, os.path.dirname(__file__))
+
+
+@hydra.main(config_path=relative_task_directory, config_name="scriptconfig")
```

Comment:
what is this `config_path=`?

Reply:
Ah, that's so the script can find the original Mephisto `parlai_chat_task_demo` YAML configuration folder despite the user calling this script from within the ParlAI repo

Reply:
that's good to know